docker-assets/auth-api/auth_api_service.rb

```diff
@@ -0,0 +1,55 @@
+require "sinatra/base"
+require "dbus"
+require "json"
+
+class AuthApiService < Sinatra::Base
+  DEFAULT_USER_ATTRIBUTES = %w(mail givenname sn displayname domainname).freeze
+
+  set :bind, "0.0.0.0"
+  set :port, ENV["HTTPD_AUTH_API_SERVICE_PORT"]
```

Comment:
Does this need to be configurable?

Reply:
Not offhand. unless they really, really want to change it, they can update it in the template and redeploy.

Reply:
I think we should be setting this to `3301` directly. Additionally I don't see it defined in the deployment env section in https://github.com/ManageIQ/manageiq-pods/pull/204 so I'm not sure how this would work as is.

Reply:
Is 3301 the default port for Sinatra?

Reply:
This env variable is provided automatically based on the service presence in the namespace (see: https://docs.openshift.org/latest/dev_guide/environment_variables.html#automatically-added-environment-variables)

Reply:
3301 is not the Sinatra port. It's an unused port as per IANA.

Reply:
Why aren't we using the default port?

Reply:
I didn't want to use the default port,  If another service comes along and uses Sinatra (though unlikely, but you never know), then we'd have to reconfigure.  I'd rather have dedicated ports for the different services we implement.

Reply:
> If another service comes along and uses Sinatra

Another sinatra app running in this container seems exceedingly unlikely.

Reply:
How about 8080